SEED-platform/seed
# Refactor: Show spinner utility during inventory export

#### Any background context you want to provide?

While exporting a file from the Inventory List to CSV there is no indication that the program is working. 

#### What's this PR do?

This PR adds the spinner utility to the screen while the export is in progress. During the export the buttons on the modal are disabled. Following a successful export, the spinner is hidden. 

#### How should this be manually tested?

Navigate to Inventory, select all properties, go to actions, click Export Selected, add a name, and click any of the export options. 
Before the file is downloaded the spinner utility will appear and buttons will be disabled. Once the download begins the spinner utility will be hidden.

#### What are the relevant tickets?
See #2524

#### Screenshots (if appropriate)

https://user-images.githubusercontent.com/58446472/139335396-3e2e15b6-e7f1-4ace-861e-6d2f1dab15cb.mov
